### PR TITLE
Add some fucking class to #a

### PR DESCRIPTION
**Pull request by mbulmer**
Needs more Benedict Cumberbatch, also testing importing JS libs.

### CODE REVIEW

File: scripts/cumberbatch.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#   None
+#
+# Commands:
+#   cumberbatch - Creates your very own Benedict Cumberbatch
+#
+# Author:
+#   mbulmer
+
+module.exports = (robot) ->
+  robot.respond /(cumberbatch)/i, (msg) ->
```
Comment (Drewzar) [inline scripts/cumberbatch.coffee]: Why not `robot.hear` ?

Comment (mbulmer) [inline scripts/cumberbatch.coffee]: Similar scripts used robot.respond so I followed suit. Is robot.hear more appropriate for these?

File: lib/cumberbatch.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  var random2 = parseInt(Math.random() * lastnamelist.length);
+  var randomfull = parseInt(Math.random() * fullnamelist.length);
+
+  var name = firstnamelist[random1] + " " + lastnamelist[random2];
+  var fullname = fullnamelist[randomfull];
+
+  var numberroll = Math.floor(Math.random() * 10) + 1;
+  var finalname = "";
+
+  return (numberroll > 9) ? fullname : name;
```
Comment (drobati) [inline lib/cumberbatch.js]: So one-tenth the time it uses a random name from the fullnamelist, the other nine-tenths it combines a first and last name randomly?

File: scripts/cumberbatch.coffee (hunk 1 above)
Comment (Drewzar) [inline scripts/cumberbatch.coffee]: I wouldn't say anything we do is **appropriate** but the robot would fire the script whenever it hears anyone say `/(cumberbatch)/i` regardless of being called directly. 

See: https://github.com/github/hubot/blob/master/docs/scripting.md#hearing-and-responding

**Pull request merged**